huaweicloud/services/acceptance/iotda/resource_huaweicloud_iotda_device_group_test.go

```diff
@@ -72,6 +72,59 @@ func TestAccDeviceGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccDeviceGroup_derived(t *testing.T) {
```

Comment:
It seems that this new use case does not use the environment face change HW_IOTDA_ACCESS_ADDRESS.